packages/flutter/lib/src/widgets/form.dart

```diff
@@ -235,8 +241,22 @@ class FormState extends State<Form> {
 
   bool _validate() {
     bool hasError = false;
+    String errorMessage = '';
     for (final FormFieldState<dynamic> field in _fields) {
       hasError = !field.validate() || hasError;
+      errorMessage += field.errorText ?? '';
+    }
+
+    if(hasError) {
```

Comment:
what happen if the errorMessage is empty? Does the native textfield announce anything in this case?

Reply:
Haven't check the native behavior but it seems make sense to not announce.